test/plugin/plugin.js

```diff
@@ -14,18 +14,18 @@ const expected_webpack_assets =
 {
 	"javascript":
 	{
-		"main": "/assets/main.6c2b37c0fc8c0592e2d3.js"
+		"main": "http://127.0.0.1:3001/assets/main.6c2b37c0fc8c0592e2d3.js"
```

Comment:
I think this [webpack-stats.json](https://github.com/halt-hammerzeit/webpack-isomorphic-tools/blob/master/test/plugin/webpack-stats.json.js) can't be generated when set the `output.publicPath` to `/assets/`. So I modified this test file.